samples/Mvc.Client/Startup.cs

```diff
@@ -42,6 +43,15 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClientId = "49e302895d8b09ea5656";
                 options.ClientSecret = "98f1bf028608901e9df91d64ee61536fe562064b";
                 options.Scope.Add("user:email");
+            })
+            
+            .AddShopify(options =>
+            {
+                options.ApiKey = "9489b7555256142d9d723711c4614df7";
```

Comment:
Would you be OK with your keys living in the sample and being used by potentially anyone?